docs/getting_started.md

```diff
@@ -25,6 +25,8 @@ Please refer to [data_preparation.md](./preprocess_dataset.md) for data preparat
 ## Body Model Preparation
 
 - [SMPL](https://smpl.is.tue.mpg.de/) v1.0 is used in our experiments.
+  - Neutral model can be downloaded from [SMPLify](https://smplify.is.tue.mpg.de/).
+  - All body models have to be renamed in `SMPL_{GENDER}.pkl` format.
```

Comment:
Should we give a script or a  code snippet? e.g., `mv basicModel_f_lbs_10_207_0_v1.1.0.pkl SMPL_FEMALE.pkl`

Reply:
Fixed in https://github.com/open-mmlab/mmhuman3d/pull/21/commits/972158d319c3f14a6129f4b7e692df6f3d1ccea5.